pom.xml

```diff
@@ -0,0 +1,200 @@
+    <packaging>jar</packaging>
+
+    <name>${project.groupId}:${project.name}</name>
+    <description>
+        Put a short paragraph-sized summary of the project here.
+    </description>
+    <!-- Check the calendar -->
+    <inceptionYear>2021</inceptionYear>
+    <url>https://nasa-pds.github.io/${project.name}</url>
+    <issueManagement>
+        <system>GitHub</system>
+        <url>https://nasa-pds.github.io/${project.name}/issues</url>
+    </issueManagement>
+    <scm>
+        <connection>scm:git:https://github.com/NASA-PDS/${project.name}.git</connection>
+        <developerConnection>scm:git:ssh://github.com/NASA-PDS/${project.name}.git</developerConnection>
+        <url>https://nasa-pds.github.io/${project.name}</url>
+        <tag>main</tag>
+    </scm>
+
+    <!-- Put the packages your project depends on here -->
+    <dependencies>
+        <!-- If you do any kind of testing—and you should—use Junit, which is the gold standard for Java -->
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter</artifactId>
+            <version>5.8.0-M1</version>
+            <scope>test</scope>
+        </dependency>
+        <!--
+        Here's an example dependency; you can find dependencies at https://search.maven.org. This isn't
+        actually used by this template, but is shown full-fleshed out because chances are your project
+        will need at least one if not many dependencies.
+        -->
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-numbers-primes</artifactId>
+            <version>1.0</version>
+            <type>jar</type>
+            <scope>compile</scope>
+            <optional>false</optional>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <!--
+        Resources (files) are non-code parts of a project that the code-parts access during run time; these
+        should be bundled in a jar (or war, or whatever) and get specified here
+        -->
+        <resources>
+            <resource>
+                <directory>${basedir}/src/main/resources</directory>
+                <filtering>true</filtering>
+                <includes>
+                    <include>example.properties</include>
+                </includes>
+                <targetPath>gov/nasa/pds/mypackage</targetPath>
+            </resource>
+            <!-- You can give <testResources> for resources needed during the test phase -->
+        </resources>
+
+        <!--
+        Plugins tell Maven how to operate the various lifecycle phases; include extra as needed, such
+        as for CORBA stub generation from `.idl` files, or even more modern things.
+        -->
+        <plugins>
+            <!-- This plugin actually compiles the `.java` source code, so you probably need it! -->
+            <plugin>
+                <artifactId>maven-compiler-plugin</artifactId>
```

Comment:
@nutjob4life if we use pds parent, then i don't think we need this?

Reply:
Included in 52b4149f5be360ad91b83281190013aa681a9e55